test/special-characters.js

```diff
@@ -247,38 +246,34 @@ describe('special characters', () => {
 
   describe('path characters', () => {
     it('should match windows drives with globstars', () => {
-      assert(isMatch('bar/', '**'));
-      assert(isMatch('A://', '**'));
-      assert(isMatch('B:foo/a/b/c/d', '**'));
-      assert(isMatch('C:/Users/', '**'));
-      assert(isMatch('c:\\', '**'));
-      assert(isMatch('C:\\Users\\', '**'));
-      assert(isMatch('C:cwd/another', '**'));
-      assert(isMatch('C:cwd\\another', '**'));
+      assert(isMatch('bar/', '**', { windows: true }));
+      assert(isMatch('A://', '**', { windows: true }));
+      assert(isMatch('B:foo/a/b/c/d', '**', { windows: true }));
+      assert(isMatch('C:/Users/', '**', { windows: true }));
+      assert(isMatch('c:\\', '**', { windows: true }));
+      assert(isMatch('C:\\Users\\', '**', { windows: true }));
+      assert(isMatch('C:cwd/another', '**', { windows: true }));
+      assert(isMatch('C:cwd\\another', '**', { windows: true }));
     });
```

Comment:
These are wrong. The entire point was to test these using platform detection, and without `windows: true`

Reply:
Close all accounts